config/src/main/java/org/hyperledger/besu/config/CliqueConfigOptions.java

```diff
@@ -59,13 +66,40 @@ public int getBlockPeriodSeconds() {
         cliqueConfigRoot, "blockperiodseconds", DEFAULT_BLOCK_PERIOD_SECONDS);
   }
 
+  public Optional<String> getBuilderApiEndpoint() {
+    String endpoint = JsonUtil.getString(cliqueConfigRoot, "builderapiendpoint", "");
```

Comment:
Wow, why so low ;)
```suggestion
    String endpoint = JsonUtil.getString(cliqueConfigRoot, "builderApiEndpoint", "");
```

Reply:
if you see the clique config it is their convention to set all params to lower case